website/files/models/base.py

```diff
@@ -167,6 +168,10 @@ def get_guid(self, create=False):
                 return None
         return Guid.generate(self)
 
+    # used by django and DRF
+    def get_absolute_url(self):
```

Comment:
We've been using this pattern other places. Not saying it's the best pattern but it's a decent idea to keep it consistent.

```python
    @property
    def absolute_api_v2_url(self):
        return absolute_reverse('nodes:node-detail', kwargs={'node_id': self._id})

    # used by django and DRF
    def get_absolute_url(self):
        return self.absolute_api_v2_url
```

I believe there are some things that call the absolute_api_v2_url directly.